data/goruco/goruco-2017/videos.yml

```diff
@@ -51,9 +51,9 @@
   video_id: 2C_IUMYzM7A
 
 - title: Developer Productivity Engineering
-  raw_title: 'GORUCO 2017: Developer Productivity Engineering by  Panayiotis Thomakos'
+  raw_title: 'GORUCO 2017: Developer Productivity Engineering by Pan Thomakos'
   speakers:
-    - Panayiotis Thomakos
+    - Pan Thomakos
```

Comment:
We have the concept of "Canonical Speakers". I think we should setup one for "Panayiotis Thomakos" that sets the canonical to "Pan Thomakos" .